### ISSUE TITLE
Design Web Literacy page

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.webmaker.org#52]
### RACI
* Phase: Design
* Owner: @hannahkane 
* Decision Maker: @hannahkane 
* Design Lead: @iamjessklein
* Dev Lead: NA
* Quality: @davidascher 
* Consulted: @dajbelshaw 

Headline: "What is Web Literacy?"
Subhead: "The skills and competencies needed for reading, writing and participating on the web."

Primary content: 
* latest version of competency grid  (currently at https://webmaker.org/resources, but remove links to the individual resource pages)
* the full set of strands, competencies and skills that comprise the Web Literacy Map (currently at https://webmaker.org/literacy)

(Important! Both of the above will have some small revisions based on @dajbelshaw's work with the community. Final revisions to come by the end of the heartbeat.)

Secondary content: “Read more about Mozilla's mission [link to manifesto] and why we care about web literacy [link to https://mozilla.github.io/webmaker-whitepaper/]"

Tertiary content: "Want to help develop the Web Literacy Map?" [link to https://wiki.mozilla.org/Webmaker/WebLiteracyMap]

**Comment by dajbelshaw** [comment]
Hey @hannahkane could you add me as 'consulted' for this one please? :)

**Comment by dajbelshaw** [comment]
Hey @hannahkane, have added some thoughts here: https://teach.etherpad.mozilla.org/weblitpage

**Comment by hannahkane** [comment]
Thanks, @dajbelshaw. I added some comments/questions to the etherpad, and updated the ticket description.

Couple questions:
 - Is there a single Discourse thread we can point to about the Web Literacy Map?
 - Is v1.5 ready to go? If so, can you please list the changes in this ticket?

**Comment by dajbelshaw** [comment]
Great, thanks @hannahkane. Three points to make:

1. I think we should link either to the [wiki page](https://wiki.mozilla.org/Webmaker/WebLiteracyMap) or [GitHub repo](https://github.com/mozilla/webliteracymap/) for ongoing development/getting involved. The wiki is probably the least scary. The problem with linking to forums is that it's difficult to grok what's going on (and there's inevitable periods of inactivity).

2. We need to link to the Mozilla Manifesto for a couple of reasons. First, as part of the process for scoping out v1.5 we decided to include a link to the manifesto instead of explicitly aligning all of the competencies to it. Second, it's a good way to demonstrate the values inherent in the Map.

3. We still need to [nail down the skills](https://github.com/mozilla/webliteracymap/issues/19), but the competencies are ready to go:

EXPLORING
*Reading the web*
* Navigation
* Web Mechanics
* Search
* Credibility
* Security

BUILDING
*Writing the web*
* Composing for the web
* Remixing
* Designing for the web
* Coding / Scripting
* Accessibility

CONNECTING
*Participating on the web*
* Sharing
* Collaborating
* Community Participation
* Privacy
* Open Practices

We'll finalise the competency descriptors on next week's call and then the skills should be ready the week after. I'm assuming that gives us enough time for localisation? Let me know if not. :)

**Comment by hannahkane** [comment]
Thanks, @dajbelshaw. Added the link to the wiki to the ticket description, and noted that the final skill lists are pending.

What about linking to the Mozilla Manifesto from the wiki page? I want to keep this page as simple as possible, and wonder about how critical the Manifesto is at this level.

**Comment by dajbelshaw** [comment]
+1 to keeping the page as simple as possible, @hannahkane. We decided to link to the manifesto in the surrounding text rather than explicitly base this revision of the Web Literacy Map upon it.

Perhaps it's as simple as extending the current text at [webmaker.org/literacy](https://webmaker.org/literacy)?

> Read more about [Mozilla's mission](https://www.mozilla.org/en-US/about/manifesto/) and [why we care about web literacy]((https://mozilla.github.io/webmaker-whitepaper/).

Does that work? :)

**Comment by hannahkane** [comment]
Ah, understood. Updated ticket description with that copy. Thanks!

**Comment by cassiemc** [comment]
@iamjessklein can you help wireframe this one out?

**Comment by iamjessklein** [comment]
I can tackle this after I complete the other two tickets I'm on.

**Comment by hannahkane** [comment]
@iamjessklein and I squared up on this today. Excited to see this page!

**Comment by iamjessklein** [comment]
@dajbelshaw - have the skills been nailed down at this point?

**Comment by jgmac1106** [comment]
@iamjessklein We hope to finalize the skills in today's community call. If
we don't finish (I think we will) a hack session is planned for Monday.

All of the issues in git have been updated with the skills still under
review and the questions revolving around those skills.

**Comment by iamjessklein** [comment]
thanks for the update @jgmac1106 - can you link me to each of the relevant issues here?

**Comment by jgmac1106** [comment]
This is the meta issue that links to all the issues

https://github.com/mozilla/webliteracymap/issues/36

You can click on each competency. I spent the last few days going between the spreadsheet and the issues adding the last bit of comments. 

Hoping we bang out the skills today.

**Comment by iamjessklein** [comment]
thanks @jgmac1106 :+1:  can you please ping this issue with an update when that's done?

**Comment by dajbelshaw** [comment]
Thanks @jgmac1106 and sorry @iamjessklein, was on PTO.

You've got the meta issue linking to all of the issues - as you can see, there's a few things we need to sort out.

Is this a blocker for further work, or just a nice to have at this point?

**Comment by iamjessklein** [comment]
@dajbelshaw - this will be in the next heartbeat. I'd like to try thinking through how we might integrate the competencies and skills in the visualization. Ideally, this will be locked down before starting the next heartbeat.

**Comment by dajbelshaw** [comment]
Great! I think given that it's a point revision (i.e. v1.5 rather than v2.0) we should keep the grid as the first point of contact. However, the interaction design could be much more radical than we've seen previously. :)

**Comment by thornet** [comment]
To help out the design process by having all the text on one page, I've copied the [latest version](https://github.com/mozilla/webliteracymap/issues/36) of the map into: 

https://teach.etherpad.mozilla.org/weblitpage
line 21

@iamjessklein that should help with designing the page! 

Just a note that there will be a few copy edits before the end of the heartbeat. @dajbelshaw can notify you of those. I've kept the pending areas in color in the etherpad so they're easy to spot.

**Comment by dajbelshaw** [comment]
These will be locked down by 4pm GMT tomorrow. Once they're all ticked off at the following URL then that's the skills and competencies locked down for v1.5!

https://github.com/mozilla/webliteracymap/issues/36

**Comment by iamjessklein** [comment]
I took a first stab at the wireframes (please provide feedback here on invision) [http://invis.io/6R2HHCS4M] 

![webliteracy](https://cloud.githubusercontent.com/assets/535012/6733661/e8b6b46e-ce29-11e4-9a46-2ca5e1031654.png)

**Comment by iamjessklein** [comment]
Tagging @vazquez to start the UI work.

**Comment by hannahkane** [comment]
I think this looks good. @dajbelshaw and @LauraHilliger have commented. @chrislarry33 - do you want to take a look at [http://invis.io/6R2HHCS4M] ?

**Comment by dajbelshaw** [comment]
WEB LITERACY MAP v1.5 FINISHED

* https://github.com/mozilla/webliteracymap
* http://literaci.es/web-literacy-map-v1-5

![cheers - dicaprio](https://cloud.githubusercontent.com/assets/2429426/6756661/36cfb296-cf23-11e4-974b-be43f2ccce8b.gif)

**Comment by vazquez** [comment]
UI Review @cassiemc http://invis.io/RQ2HY7A7G

**Comment by iamjessklein** [comment]
nice job @vazquez

**Comment by cassiemc** [comment]
Ok done @vazquez just a few things to address! Really close, layout / organization and treatments look great, just some spacing / alignment issues at most.

**Comment by vazquez** [comment]
Addressed feedback, another round of UI Review @cassiemc http://invis.io/RQ2HY7A7G

**Comment by cassiemc** [comment]
Looks great @vazquez, just one little spacing issue on the bullets in the web lit dropdowns. Otherwise R+

**Comment by vazquez** [comment]
Final feedback addressed. Design can move to coding @hannahkane 

http://invis.io/RQ2HY7A7G

:boom:

**Comment by hannahkane** [comment]
Awesome! Unblocked #51.
